src/gui/qgsmaptooledit.h

```diff
@@ -67,12 +67,26 @@ class GUI_EXPORT QgsMapToolEdit: public QgsMapTool
     //! Returns the current vector layer of the map canvas or 0
     QgsVectorLayer *currentVectorLayer();
 
+    //! Result of addTopologicalPoints
+    enum TopologicalResult
+    {
+      Success = 0, //!< AddTopologicalPoints was successful
+      InvalidCanvas = 1, //!< AddTopologicalPoints failed due to an invalid canvas
+      InvalidLayer = 2, //!< AddTopologicalPoints failed due to an invalid canvas
+    };
+
+    /**
+     * Adds vertices to other features to keep topology up to date, e.g. to neighbouring polygons.
+     * \param geom list of points (in layer coordinate system)
```

Comment:
\since 3.10

I'd also suggest reworking this to avoid the "geom" abbreviation, e.g.

/**
     * Adds a list of \a vertices to other features to keep topology up to date, e.g. to neighbouring polygons.
     * The \a vertices list specifies a set of points topological points to add, in the layer's coordinate reference system.
     * \since QGIS 3.10
*/
TopologicalResult addTopologicalPoints( const QVector<QgsPointXY>& vertices );